src/views/registration-js/attendees/fields/checkbox.php

```diff
@@ -31,7 +26,8 @@
 ?>
 <div class="tribe-field tribe-tickets-meta-fieldset tribe-tickets-meta-fieldset__checkbox-radio <?php echo $required ? 'tribe-tickets-meta-required' : ''; ?>">
 	<header class="tribe-tickets-meta-label">
-		<h3 class="tribe-common-b1 tribe-common-b2--min-medium"><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></h3>
+		<h3 class="tribe-common-b1 tribe-common-b2--min-medium"><?php echo wp_kses_post( $field['label'] ); ?>
+		<?php tribe_required_label( $required ); ?></h3>
```

Comment:
this indentation looks weird, was there a reason why it was done like this?

Reply:
@paulmskim thanks for the review! Yes, these templates are the current templates and we're deprecating them in favor of the new ET+ ones with introduced with RSVP V2: https://github.com/moderntribe/event-tickets-plus/tree/master/src/views/v2/components/meta - These new have snapshot tests and everything is in place. The intention of this ticket is to fix this noticeable issue for the current users before we move to the new ones.

No particular reason for having the required on a new line. Would you prefer to have everything in one line or put the text + required in new lines and indent from the `h3`?

Reply:
OK, great! Yeah, I was thinking either all on one line or the way you have it now.

If there are snapshot tests in place for this template, shouldn't we have snapshots update because we've made spacing changes to the template?

Reply:
There are no snapshot tests for these templates. They are going to be deprecated. 

There are tests on the new ones in ET+, which are the ones we'll start using on the new "tickets v2" and which we already started using in RSVP v2. https://github.com/moderntribe/event-tickets-plus/tree/master/tests/wpunit/Tribe/Tickets_Plus/Partials/V2/Components/Meta

Reply:
Oh gotcha, misunderstood your first comment. Thanks!

Reply:
Thanks, and sorry if it was confusing!